File: AP1/Estado.cpp
#include<stdio.h>
#include<conio.h>

main()
{
	char codigo;
	
	printf("\nDigite a letra para o seu estado: ");
	printf("\nDigite (r) para Rio Grande do Sul");
	printf("\nDigite (s) para Santa Catarina");
	printf("\nDigite (p) para Parana");
	scanf("%c",&codigo);
	
	if((codigo == 'r'))
		printf("\nGaucho!");
		{
			if((codigo=='s'))
			printf("\nCatarina!");
		}
		{
			if((codigo=='p'))
			printf("\nParanaense!");
		}
		{
			if((codigo!='r')&&(codigo!='p')&&(codigo!='s'))
			printf("\nEstado nao identificado!");
		}
}

File: IAC/Estado.cs
using System;

class Estado
{
    public static void Main()
    {
        int cod=0;

        do
        {
            if(!(cod==1)||(cod==2)||(cod==3)){
            Console.WriteLine("Digite o número para seu estado:");
            Console.WriteLine("Digite (1) para Rio Grande do Sul:");
            Console.WriteLine("Digite (2) para Santa Catarina:");
            Console.WriteLine("Digite (3) para Paraná:");
            cod = int.Parse(Console.ReadLine());
            }
            if(cod == 1)
            {
                Console.Clear();
                Console.WriteLine("Gaúcho!");
                break;
            }  
            if(cod == 2)
            {
                Console.Clear();
                Console.WriteLine("Catarinense!");
                break;
            }  
            if(cod == 3)
            {
                Console.Clear();
                Console.WriteLine("Paranaense!");
                break;
            }
            if((cod != 1)&(cod != 2)&(cod != 3)) 
            {
                Console.Clear();
                Console.WriteLine("Estado não identificado!");
            }
            
        }while((cod != 1)||(cod != 2)||(cod != 3));
        Console.ReadLine();
    }
}
